internal/app/grpc/interceptor.go

```diff
@@ -0,0 +1,93 @@
+package grpc
+
+import (
+	"context"
+	"github.com/korol8484/shortener/internal/app/usecase"
+	"github.com/korol8484/shortener/internal/app/user/util"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+	"net"
+	"slices"
+	"strings"
+)
+
+// JwtInterceptor - add or read auth info from request
+func JwtInterceptor(jwt *usecase.Jwt) grpc.UnaryServerInterceptor {
+	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
```

Comment:
Я советую использовать ключевое слово `any`  вместо `interface{}`. 
```suggestion
	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (any, error) {
```